### DISCUSSION TITLE
MakeCode arcade

### DISCUSSION THREAD

**Discussion by zAlucard** [jtothebell/fake-08#210] [Ideas]
I was wondering if this emulator could be setup to run MakeCode Arcade PNG files. I tried but I got a resolution error. MakeCode uses 160 x 120 x 4 bits.

I have a 7 year old who is just starting in programming with MakeCode (Code Ninjas uses a variation of it) and I know it would be a huge boost to focus if he could get his games on a Miyoo Mini.

**Comment by DPS2004** [comment]
I think MakeCode Arcade is entirely separate from PICO-8, considering it uses Javascript and is developed by Microsoft.

**Comment by jtothebell** [comment]
Yeah, I had never heard of MakeCode arcade before this post. Unfortunately I don't think there would be much crossover in this project.
